definitions/divorce/json/CaseEventToFields/CaseEventToFields.json

```diff
@@ -10294,5 +10294,17 @@
     "PageID": "ReturnToPreviousState1",
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1
+  },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
I know it's ignored by CCD, but it's easier to investigate when you use current date

Reply:
Addressed in 4b25a21f5bf186026c7933e86f859a8449132a4f